addons/crm/__manifest__.py

```diff
@@ -46,6 +46,7 @@
 
         'views/calendar_views.xml',
         'views/crm_recurring_plan_views.xml',
+        'views/res_partner_views.xml',
         'views/crm_menu_views.xml',
```

Comment:
Would prefer to set crm_menu_views as last loaded file, as it depends on actions defined before anyway.